conf/nginx.conf.tpl

```diff
@@ -81,7 +92,16 @@ http {
     }
 
     location / {
-      ${FALLBACK_DIRECTIVES}
+      proxy_pass $root_url;
```

Comment:
Duplicating this from the `location` directive above means that once we switch the marketing website we can just get rid of that directive.